pkg/apis/node/v1alpha1/types.go

```diff
@@ -278,3 +282,42 @@ type CustomNodeResourceList struct {
 	// items is the list of CNRs
 	Items []CustomNodeResource `json:"items"`
 }
+
+// NodeMetricStatus defines the observed state of NodeMetric
+type NodeMetricStatus struct {
+	// UpdateTime is the last time this NodeMetricStatus was updated.
+	UpdateTime metav1.Time `json:"updateTime"`
+
+	// NodeMetric contains the metrics for this node.
+	NodeMetric *NodeMetricInfo `json:"nodeMetric,omitempty"`
+
+	// GroupMetric contains the metrics aggregated by QoS level groups
+	GroupMetric []GroupMetricInfo `json:"groupMetric,omitempty"`
+}
+
+type ResourceUsage struct {
+	// NUMAUsage contains the real-time resource usage for each NUMA
+	NUMAUsage []NUMAMetricInfo `json:"numaUsage,omitempty"`
+
+	// GenericUsage contains the real-time resource usage, in format of '{"cpu": 2, "memory": "1Gi"}'
```

Comment:
since we already have `NUMAUsage` why we still need `GenericUsage`? should we restrict that only one of these two fields is valid?

Reply:
Not all metrics can be granularized to the NUMA level，such as network bandwidth.

Reply:
ok, so we should restrict that only one of these two fields is valid (for a certain resource)? if so, pls add some comments here